app/models/post.rb

```diff
@@ -5,6 +5,7 @@ class Post < ActiveRecord::Base
   has_many :votes, dependent: :destroy
 
   default_scope { order('created_at DESC') }
+  default_scope { order('rank DESC') }
 
```

Comment:
A model may only have one default_scope. Review lines 7 and 8. 

It is possible to order by multiple columns, for example:
```
# Orders by rank 1st, then created_at 2nd. If 2 posts have the same rank
# then the newer post would be ordered before the older post (due to 'created_at DESC'):
default_scope { order('rank DESC, created_at DESC') }
```
though I'm not sure if this is what's intended.